Add VMWare Tanzu notes. Also differentiate between supported and tested k8s (#719)
stackabletech/documentation: modules/ROOT/pages/kubernetes/vmware_tanzu.adoc

= VMware Tanzu

VMware Tanzu uses a non-standard Kubelet state directory.

> Therefore installing secret-operator and listener-operator on VMware Tanzu requires a special treatment.
> Please read on the xref:secret-operator:installation.adoc#_vmware_tanzu[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_vmware_tanzu[listener-operator installation guide] for details.

Reviewer: ```suggestion
Therefore, installing the secret-operator and listener-operator on VMware Tanzu requires special handling.
Please refer to the xref:secret-operator:installation.adoc#_vmware_tanzu[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_vmware_tanzu[listener-operator installation guide] for details.
```
